frontend/src/components/Buttons/Playback/PlaybackSettingsButton.vue

```diff
@@ -8,8 +8,7 @@
       <tooltip-button
```

Comment:
Agree with this since the component is really basic and doesn't have any real special/custom implementation. If we're all aligned on that being the way forward I can add it to this PR or do it separately. Will wait until we have consensus.